### ISSUE TITLE
include-v-in-tag: false 

### ISSUE THREAD

**Issue by MHuiG** [google-github-actions/release-please-action#558]
### TL;DR

If include-v-in-tag: false is configured, the version number still contains v

### Expected behavior

version: 5.6.0

https://github.com/MHG-LAB/hexo-theme-volantis/tree/5.6.0

### Observed behavior

version: v5.6.0

https://github.com/MHG-LAB/hexo-theme-volantis/tree/v5.6.0

### Action YAML

https://github.com/MHG-LAB/hexo-theme-volantis/commit/208d7a329de8a9dcf30d65e794c5c43315c2e72a

### Log output

https://github.com/MHG-LAB/hexo-theme-volantis/runs/7822163600?check_suite_focus=true

### Additional information

doc:

include-v-in-tag	include "v" in tag versions. Default true

logs:

Warning: Unexpected input(s) 'include-v-in-tag',

Is the document out of date？

**Comment by chingor13** [comment]
This option is being added in v3.4.0 (#474). I've been having trouble releasing it because of default value issues.

**Comment by chingor13** [comment]
3.4.0 should have fixed this now
